pkg/github/projects.go

```diff
@@ -237,8 +239,9 @@ func ListProjectFields(t translations.TranslationHelperFunc) inventory.ServerToo
 			Name:        "list_project_fields",
 			Description: t("TOOL_LIST_PROJECT_FIELDS_DESCRIPTION", "List Project fields for a user or org"),
 			Annotations: &mcp.ToolAnnotations{
-				Title:        t("TOOL_LIST_PROJECT_FIELDS_USER_TITLE", "List project fields"),
-				ReadOnlyHint: true,
+				Title:         t("TOOL_LIST_PROJECT_FIELDS_USER_TITLE", "List project fields"),
```

Comment:
Similar question I think fields also only created by project admins and not sure public projects exist